src/flux/reference-book.coffee

```diff
@@ -25,10 +25,19 @@ ReferenceBookConfig = {
     getToc: (courseId) ->
       @_get(courseId)['0']
 
-    getChapterFirstPage: (courseId, chapterId) ->
+    # Takes a courseId and a chapter_section specifier
+    # which is a string joined with dots i.e. "1.2.3"
+    getChapterSectionPage: ({courseId, section}) ->
+      parts = _.map(section.split('.'), (part) -> parseInt(part, 10) )
```

Comment:
Not really since it's a flux store and doesn't do any formatting.  What it does is attempt to find the first first page for a given section.